pretyping/nativenorm.ml

```diff
@@ -379,6 +435,52 @@ let evars_of_evar_map sigma =
     Nativelambda.evars_typ = Evd.existential_type sigma;
     Nativelambda.evars_metas = Evd.meta_type sigma }
 
+(* fork perf process, return profiler's process id *)
+let start_profiler_linux () =
+  let coq_pid = Unix.getpid () in (* pass pid of running coqtop *)
+  (* we don't want to see perf's console output *)
+  let dev_null = Unix.descr_of_out_channel (open_out_bin "/dev/null") in
+  let profile_fn = get_available_profile_filename () in
```

Comment:
Well, if you use `native_compute` under `par:` I guess it could be problematic in theory. One way of fixing it would be to use `Filename.open_temp_file`.